frontend/src/app/utils/disease.ts

```diff
@@ -0,0 +1,12 @@
+import { useFeature } from "flagged";
+
+import { MULTIPLEX_DISEASES } from "../testResults/constants";
+
+export const useSupportedDiseaseList = () => {
+  let allDiseases = Object.values(MULTIPLEX_DISEASES);
+  const hivEnabled = Boolean(useFeature("hivEnabled"));
```

Comment:
dev6 had `hivEnabled` set as true so for this PR I manually set that flag to false using the UpdateFeatureFlag mutation